great_expectations/render/types/__init__.py

```diff
@@ -1,14 +1,47 @@
+from __future__ import annotations
+
 import json
 from copy import deepcopy
+from enum import Enum
 from string import Template as pTemplate
-from typing import List, Optional
+from typing import List, Optional, Union
 
 from marshmallow import INCLUDE, Schema, fields, post_dump, post_load
 
 from great_expectations.render.exceptions import InvalidRenderedContentError
 from great_expectations.types import DictDot
 
 
+class AtomicRendererPrefix(Enum):
+    """Available atomic renderer prefixes"""
+
+    PRESCRIPTIVE = "atomic.prescriptive"
+    DIAGNOSTIC = "atomic.diagnostic"
+
+    def __eq__(self, other: Union[str, AtomicRendererPrefix]):
+        if isinstance(other, str):
+            return self.value.lower() == other.lower()
+        return self.value.lower() == other.value.lower()
+
+    def __hash__(self: AtomicRendererPrefix):
+        return hash(self.value)
```

Comment:
Can we add return annotations here?

Reply:
@NathanFarmer 
On a related note. Right now `render/types` is not being type-checked, but it only has a couple of typing issues.

Could you remove it from the type-check exclude list and fix (or ignore) the 4 or so issues as part of this PR.
https://github.com/great-expectations/great_expectations/blob/develop/pyproject.toml#L97

Reply:
🤦🏻 ✅